src/time_integration/paired_explicit_runge_kutta/paired_explicit_runge_kutta.jl

```diff
@@ -7,6 +7,12 @@
 
 # Basic implementation of the second-order paired explicit Runge-Kutta (PERK) method
 include("methods_PERK2.jl")
+include("methods_PERK3.jl")
 # Define all of the functions necessary for polynomial optimizations
 include("polynomial_optimizer.jl")
+
+# Add definitions of functions related to polynomial optimization by NLsolve here
+# such that hey can be exported from Trixi.jl and extended in the TrixiConvexECOSExt package
+# extension or by the NLsolve-specific code loaded by Requires.jl
+function solve_a_unknown! end
```

Comment:
Could we please use more descriptive function names? This reads like we want to solve for an arbitrary unknown but I guess you want to obtain Butcher coefficients here.

Reply:
Copy that!